lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -1807,7 +1807,7 @@ public function TicketChange($ticketid, array $props)
             $props['requestor_mail'] = $ticket['requestor_mail'];
         }
 
-        if ($props['priority'] == '') {
+        if ($props['priority'] == '' || !isset($props['priority'])) {
```

Comment:
To może:
```php
if (empty($props['priority'])) {
```
?

Reply:
empty nie możebyć bo 0 to priorytet normalny

Reply:
@chilek zobacz zrobiłem test.php https://gist.github.com/interduo/ec0ab75d99d9140045bc8e2850328488 daje 

$ php  test.php daje:
zmienna11 zmienna21 zmienna31 zmienna4 zmienna51

I tutaj również gratisowe jedynki powinny być tylko przy 2 i 5.

Reply:
empty() różni się znacząco od isset(). Np. empty("0") zwróci true, mimo, że ciąg tekstowy jest niepusty. empty("") również true. Nie sprowadzaj empty() do isset().

Reply:
To co zaproponowałeś spowodowałoby, że dla priorytetu normalnego ustawiałby się zawsze niezdefiniowany.

Reply:
Nie bardzo rozumiem dlaczego mój wariant testowy jest zły.